Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -22,34 +22,10 @@ public func == (lhs: HockeyConfigData, rhs: HockeyConfigData) -> Bool {
     && lhs.userName == rhs.userName
 }
 
-public protocol NotificationAuthorizationStatusType {
-  var isAuthorized: Bool { get }
-  var isDenied: Bool { get }
-  var isNotDetermined: Bool { get }
-}
-
-@available(iOS 10.0, *)
-extension UNAuthorizationStatus: NotificationAuthorizationStatusType {
-  public var isAuthorized: Bool {
-    switch self {
-    case .authorized: return true
-    case .denied, .notDetermined: return false
-    }
-  }
-
-  public var isDenied: Bool {
-    switch self {
-    case .denied: return true
-    case .authorized, .notDetermined: return false
-    }
-  }
-
-  public var isNotDetermined: Bool {
-    switch self {
-    case .notDetermined: return true
-    case .denied, .authorized: return false
-    }
-  }
+public enum NotificationAuthorizationStatusType {
+  case authorized
+  case denied
+  case notDetermined
 }
```

Comment:
We can just model this enum after the `UNAuthorizationStatus` enum and convert it with a function later. See below.